### PR TITLE
Release v3.7.6

### PR DESCRIPTION
**Pull request by medikoo**

### CODE REVIEW

File: test/unit/lib/plugins/aws/invoke-local/index.test.js [hunk 1]
```diff
@@ -997,7 +998,14 @@ describe('test/unit/lib/plugins/aws/invokeLocal/index.test.js', () => {
             },
           },
         });
-        const outputAsJson = JSON.parse(response.output);
+        const outputAsJson = (() => {
```
Comment (pgrzesik) [inline test/unit/lib/plugins/aws/invoke-local/index.test.js:1001]: Is there a reason why this change landed here?

Comment (medikoo) [reply test/unit/lib/plugins/aws/invoke-local/index.test.js:1001]: Yes, sorry for not providing an explanation. There was a CI test fail, where this parse failed due to not uniform JSON. This log was added to give a more clue, next time it happens

https://github.com/serverless/serverless/runs/5583504964?check_suite_focus=true

Comment (pgrzesik) [reply test/unit/lib/plugins/aws/invoke-local/index.test.js:1001]: Thanks for clarification 🙇

**Review by pgrzesik — APPROVED** [review]
looks great 👍
